ui-lib/src/components/Divider/index.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+type DividerTypes = "light" | "dark";
+
+interface Props {
+  margin?: number;
+  type?: DividerTypes; // Nb default to light grey
```

Comment:
Type shouldn't be a prop here, we want to design components in a way that the style is reactive.
In this implementation, we'll have to listen to the theme switch at the top of this component and return new props value depending on the user choice. 

By using a single key `theme.colors.palette.v2.grey.border` to reference two different values from the light and dark theme file, style is reactive by default.

Reply:
Once again, type makes sense if there are two types of divider per theme, if it's one per theme then indeed this is not needed

Reply:
Confirmed by design, you were correct.